engine/dlib/src/zip/zip.cpp

```diff
@@ -165,6 +165,7 @@ const char *zip_strerror(int errnum) {
   return zip_errlist[errnum];
 }
 
+#if defined(ZIP_USE_DEFLATE) // defold
```

Comment:
Some cleanup to remove "unused" warnings.
This will also go into a PR towards their repo eventually.